ember/app/components/password-reset-subpanel.js

```diff
@@ -0,0 +1,52 @@
+import Component from '@ember/component';
+import config from 'massbuilds/config/environment';
+import { action, computed } from 'ember-decorators/object';
+import { service } from 'ember-decorators/service';
+
+export default class extends Component {
+
+  @service ajax
+  @service notifications
+
+  constructor() {
+    super();
+
+    this.classNames = ['component', 'subpanel', 'form-subpanel', 'password-reset-subpanel'];
+
+    this.username = '';
+
+    this.errorMessage = null;
+    this.isResetting = false;
+    this.loadingText = 'Resetting Password';
+  }
+
+  @computed('username')
+  get submittable() {
+    return (this.get('username') !== '');
+  }
+
+  @action
+  resetPassword() {
+    if (this.get('submittable') && !this.get('isResetting')) {
+      this.set('isResetting', true);
+      this.set('errorMessage', null);
+
+      const email = this.get('username');
```

Comment:
email vs username, is that intentional?

Reply:
Only because it is referred to by other forms as `username`. `email` would be the ideal variable name across the app. The user session handling code was implemented on the client and server simultaneously which is why I had originally used `username` cause I didn't have a variable name to give it yet.

It should be refactored outside of this PR since it would involve touching more code than it concerns.

Reply:
#82